website/docs/docs/deploy/deployment-overview.md

```diff
@@ -3,16 +3,72 @@ title: "About deployments"
 id: "deployments"
 ---
 
-:::info Deployments
+Running dbt in production means setting up a system to run a _dbt job on a schedule_, rather than running dbt commands manually from the command line. Your production dbt jobs should create the tables and <Term id="view">views</Term> that your business intelligence tools and end users query. Before continuing, make sure you understand dbt's approach to [managing environments](/docs/collaborate/environments).
 
-This section provides an overview of the deployment types available for dbt Cloud.
-If you’re interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
+In addition to setting up a schedule, there are other considerations when setting up dbt to run in production:
 
-:::
+* The complexity involved in creating a new dbt job, or editing an existing one.
+* Setting up notifications if a step within your job returns an error code (for example, a model can't be built or a test fails).
+* Accessing logs to help debug any issues.
+* Pulling the latest version of your git repo before running dbt (that is, continuous deployment).
+* Running your dbt project before merging code into master (that is, continuous integration).
+* Allowing access for team members that need to collaborate on your dbt project.
 
-In general, the dbt Cloud application deployment models fall into two categories: **Multi Tenant** and **Single Tenant**. These deployments are hosted on infrastructure managed by dbt Labs. Both models leverage AWS infrastructure as described in the [Architecture](deployment-architecture) section.
+## Run dbt in production
 
-For more information on these deployment models view the below pages.
+If you want to run dbt jobs on a schedule, you can use dbt Cloud, Airflow, Prefect, Dagster, automation server, or Cron.
```

Comment:
Non-blocking: this reads as an exhaustive list. perhaps "tools such as dbt Cloud..."